CODEOWNERS

```diff
@@ -0,0 +1,40 @@
+### Default owners ###
```

Comment:
This file should be in `.github`

Reply:
That is [one possible location](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-file-location). The current issue, however, is that there are several references to a file in this location, currently in the [governance doc](https://github.com/godaddy/gasket/blob/master/GOVERNANCE.md).

Either location is fine. However, moving this up makes it more visible and fixes those links without needing to update them.

Reply:
I opted to leave it at the root for visibility. Thanks for the catch @SivanMehta, I totally overlooked that.